src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -220,7 +218,7 @@ private void linkToCase(User defendantUser, User anonymousCaseWorker, String let
         CaseDetails caseDetails = this.updateDefendantIdAndEmail(defendantUser, caseId, defendantId, defendantEmail);
 
         Claim claim = ccdCaseDataToClaim.to(caseDetails.getId(), caseDetails.getData());
-        jobSchedulerService.scheduleEmailNotificationsForDefendantResponse(claim);
+//        jobSchedulerService.scheduleEmailNotificationsForDefendantResponse(claim);
```

Comment:
This should be made conditional based on `ccd_aync_enabled` flag. Only link if the flag is TRUE.